Tools/container/docker_run.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+if [[ -z "${DOCKER_TAG}" ]]; then
+	# The default tag name should be hardcoded
+	# to whatever we are currently using in CI on this branch
+	TAG_NAME="2024-10-11"
+else
+	TAG_NAME="${DOCKER_TAG}"
+fi
+
+if [[ -z "${REPO_NAME}" ]]; then
```

Comment:
This isn't just "repo name", it's the image name which may or may not contain a full path.

I think it's less ambiguous to do the full thing together or nothing.